packages/quantic/docs/template/lwc-json/publish.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @overview Builds a tree-like JSON string from the LWC doclet data.
+ * @version 0.0.1 
+ */
+'use strict';
+
+const fs = require('fs');
+const paramCase = require('change-case').paramCase;
+const xmlToJson = require('xml2json').toJson;
+const dump = require('jsdoc/util/dumper').dump; 
+
+function formatType(type) {
+  return type
+    ? (type.names.length === 1 ? type.names[0] : type.names)
+    : '';
+}
+
+function formatBooleanParam(boolParam) {
+  return typeof boolParam === 'boolean' ? boolParam : ''
+}
+
+function parseFunction(element, parentNode) {
+  if (!isPublic(element)) {
+    return;
+  }
+
+  if (!parentNode.functions) {
+    parentNode.functions = [];
+  }
+
+  const thisFunction = {
+    name: element.name,
+    access: element.access || '',
+    virtual: !!element.virtual,
+    description: element.description || '',
+    parameters: [],
+    examples: []
+  };
+
+  parentNode.functions.push(thisFunction);
+
+  if (element.returns) {
+    thisFunction.returns = {
+      type: formatType(element.returns[0].type),
+      description: element.returns[0].description || ''
+    };
+  }
+
+  if (element.examples) {
+    for (let i = 0; i < element.examples.length; i++) {
+      thisFunction.examples.push(element.examples[i]);
+    }
+  }
+
+  if (element.params) {
+    for (let i = 0; i < element.params.length; i++) {
+      thisFunction.parameters.push({
+        name: element.params[i].name,
+        type: formatType(element.params[i].type),
+        description: element.params[i].description || '',
+        default: element.params[i].defaultvalue || '',
+        optional: formatBooleanParam(element.params[i].optional),
+      });
+    }
+  }
+}
+
+function isTagDefined(element, tagTitle) {
+  return element.tags?.filter((tag) => tag.title === tagTitle).length;
+}
+
+function isPublic(element) {
```

Comment:
This function could be simpllified to:

    function isPublic(element) {
      return isTagDefined(element, 'api');
    }

Reply:
🤦